server.js

```diff
@@ -33,11 +36,14 @@ function maybe_bounce(req, res, bounce) {
     }
 
     var subdomain = tldjs.getSubdomain(hostname);
-    if (!subdomain) {
+    if (!subdomain || subdomain === options.subdomain) {
```

Comment:
In regards to your option comments, I needed the options here, and below. If I made no other changes and put `opt` here, `opt` would be undefined. Hope that all makes sense.